pkg/auth/identityResponse.go

```diff
@@ -2,51 +2,90 @@ package auth
 
 import (
 	"encoding/base64"
+	"fmt"
 	"io/ioutil"
 
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog/v2"
 
+	responsetypes "github.com/liqotech/liqo/pkg/identityManager/responseTypes"
 	"github.com/liqotech/liqo/pkg/kubeconfig"
 	"github.com/liqotech/liqo/pkg/utils"
 )
 
+// AWSIdentityInfo contains the information required by a cluster to get a valied IAM-based identity.
+type AWSIdentityInfo struct {
+	AccessKeyID     string `json:"accessKeyID,omitempty"`
+	SecretAccessKey string `json:"secretAccessKey,omitempty"`
+	Region          string `json:"region,omitempty"`
+	EKSClusterID    string `json:"eksClusterID,omitempty"`
+	IAMUserArn      string `json:"iamUserArn,omitempty"`
```

Comment:
Just wondering, could these be empty? Or should they all be present if the parent structure is present?

Reply:
You are right, they have to be all present